### ISSUE TITLE
ML: model selection assignment

### ISSUE THREAD

**Issue by zhangqi444** [ocademy-ai/machine-learning#220]
Add this to Model Selection assignment 

@ninichen2911 
- https://www.kaggle.com/code/ksvmuralidhar/learning-curve-to-identify-overfit-underfit 
- https://www.kaggle.com/code/ryanholbrook/dropout-and-batch-normalization
